scripts/setup

```diff
@@ -273,10 +275,21 @@ then
     DOCUMENTS=${DATA_PATH}/documents_big.json
     ITEMS=${DATA_PATH}/items_big.json
     HOLDINGS=${DATA_PATH}/holdings_big.json
+    PERSONS=${DATA_PATH}/persons_big.json
 else
     DOCUMENTS=${DATA_PATH}/documents_small.json
     ITEMS=${DATA_PATH}/items_small.json
     HOLDINGS=${DATA_PATH}/holdings_small.json
+    PERSONS=${DATA_PATH}/persons_small.json
+fi
+
+if $LOADPERSONS
```

Comment:
As you change `$DEPLOYMENT` to `${DEPLOYMENT}` before, is that possible to do the same thing with `$LOADPERSONS`?

Reply:
ok